portfolio/src/main/webapp/script.js

```diff
@@ -174,6 +174,21 @@ function deleteData() {
   });
 }
 
+/**
+ * Loads the blobstore image storage and shows the form to submit an image.
+ */
+function loadBlobstoreAndForm() {
+  fetch('/blobstore-upload-url')
+      .then((response) => {
```

Comment:
This can be shortened to just "response => response.text()" inside the then parentheses.